docs/articles/vs-test-adapter/Adapter-Installation.md

```diff
@@ -30,10 +21,29 @@ To add it is a NuGet package, you must have an active solution, then follow thes
 4. Click 'Install'
 5. In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.
 
+## Installing With the Extension Manager
+
+The use of the vsix extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter.
```

Comment:
```suggestion
> [!WARNING]
> The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter. We recommend you try to move to nuget based adapters.
```

I'm thinking this will give it some flair.